packages/wdio-testingbot-service/src/service.ts

```diff
@@ -13,30 +22,30 @@ export default class TestingBotService {
     /**
      * gather information about runner
      */
-    beforeSession (config, capabilities) {
+    beforeSession (config: WebdriverIO.Options, capabilities: WebDriver.DesiredCapabilities) {
         this.config = config
         this.capabilities = capabilities
         this.config.user = config.user
         this.config.key = config.key
         this.tbUser = this.config.user
         this.tbSecret = this.config.key
 
-        this.isServiceEnabled = this.tbUser && this.tbSecret
+        this.isServiceEnabled = Boolean(this.tbUser && this.tbSecret)
     }
 
     /**
      * Before suite
      * @param {Object} suite Suite
     */
-    beforeSuite (suite) {
+    beforeSuite (suite: any) {
```

Comment:
Do we want to keep them untyped?

Reply:
No, this was an oversight. Will fix it.

Reply:
there are a bunch of `any` still the file which is why I commented https://github.com/webdriverio/webdriverio/pull/6032#issuecomment-716670117

Reply:
@christian-bromann Fixed most of the any. The remaining are:
- Cucumber types, which does not export any typing yet so I can't use types for `after/beforeFeature`
- Unit test, which sometimes must cast to any because explicit cast does not work